holmes/plugins/toolsets/kafka.py

```
@@ -818,6 +899,338 @@ def get_parameterized_one_liner(self, params: Dict) -> str:
+                    msg_count += 1
+
+                return StructuredToolResult(
+                    status=StructuredToolResultStatus.SUCCESS,
+                    data=yaml.dump({"messages": messages, "count": len(messages)}),
+                    params=params,
+                )
+            finally:
+                consumer.close()
+
+        except Exception as e:
+            error_msg = f"Failed to consume messages: {str(e)}"
```

Ruff (0.15.9)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)